lib/rubocop/cop/rspec/describe_class.rb

```diff
@@ -24,7 +25,7 @@ class DescribeClass < Cop
         FEATURE_PAIR = s(:pair, s(:sym, :type), s(:sym, :feature))
 
         MESSAGE = 'The first argument to describe should be the class or ' \
-                  'module being tested.'
+                  'module being tested.'.freeze
```

Comment:
Perhaps it would be better to disable that check until the issue is fixed in RuboCop? Otherwise a bunch of lines are going to be churned (by adding `.freeze` then removing in later).